theano/gpuarray/type.py

```diff
@@ -20,6 +20,7 @@
     pygpu = None
 
 _context_reg = {}
+_props_map = {}
```

Comment:
add in a comment an example of props. props is an overloaded name with many meaning.

Or better, add docstring to the new user methods from this file.